### PR TITLE
Adding mTLS options

### PR DESCRIPTION
**Pull request by fktkrt**
It contains the fixes in #4 as well.

### CODE REVIEW

File: charts/burrow/templates/burrow-configmap.yaml [hunk 1]
```diff
@@ -27,18 +27,27 @@ data:
     [client-profile.{{ $key }}]
     kafka-version={{ $value.kafkaVersion | default "0.8" | quote}}
     client-id="{{ $value.clientId | default "burrow-lagchecker" }}"
+    tls="{{ $value.tls | default "burrow-mTLS" }}"
```
Comment (Lowess) [inline charts/burrow/templates/burrow-configmap.yaml]: @fktkrt can you please make `tls=` block conditionally inserted. The default should be empty as mentioned in the docs https://github.com/linkedin/Burrow/wiki/Configuration#client-profile

File: charts/burrow/templates/burrow-configmap.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@ data:
 
     [zookeeper]
     servers={{ include "burrow.servers" .Values.burrow.config.zookeeper.servers }}
-    timeout={{ .Values.burrow.config.zookeeper.timout | default 6 }}
+    timeout={{ .Values.burrow.config.zookeeper.timeout | default 6 }}
```
Comment (Lowess) [inline charts/burrow/templates/burrow-configmap.yaml:21]: Good catch 😎

File: charts/burrow/templates/burrow-configmap.yaml [hunk 3]
```diff
@@ -27,18 +27,27 @@ data:
...
+    {{ range $key, $value := .Values.burrow.config.tls }}
+    [tls.{{ $key }}]
+    certfile="{{ $value.certFile | default "/tmp/client_cert.pem" }}"
+    keyfile="{{ $value.keyFile | default "/tmp/private_key.pem" }}"
+    cafile="{{ $value.caFile | default "/tmp/truststore.pem" }}"
+    noverify={{ $value.noverify | default false }}
     {{ end }}
 
     {{ range $key, $value := .Values.burrow.config.cluster }}
-    [cluster.{{ $value.clusterName | default $key }}]
```
Comment (Lowess) [inline charts/burrow/templates/burrow-configmap.yaml:33]: Makes sense to remove 

The main reason for that was to have K8s camelCase respected as I wanted my clusters to display as `va-prod-kafka` and wanted to use camelCase in the `values.config.cluster`

**Review by Lowess — COMMENTED** [review]
Please bump the version of `Chart.yaml`

**Comment by Lowess** [comment]
Hi @fktkrt thanks for your interest in this repo ! I left my review above.

File: charts/burrow/templates/burrow-configmap.yaml (hunk 1 above)
Comment (fktkrt) [reply charts/burrow/templates/burrow-configmap.yaml]: Yes. it's done.

**Review by Lowess — APPROVED** [review]

**Pull request merged**